packages/prettier-config/README.md

```diff
@@ -0,0 +1,14 @@
+# `@shopify/prettier-config`
+
+[![Build Status](https://travis-ci.com/Shopify/web-foundation.svg?branch=master)](https://travis-ci.com/Shopify/web-foundation)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](../../LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)  [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg) 
+
+Shared prettier configuration
+
+## Installation
+
+```bash
+$ yarn add @shopify/prettier-config
+```
+
+## Usage
```

Comment:
Can we fill this out, please? It might be worth adding notes on how this package is intended to be used with `@shopify/eslint-plugin`. Most of it is "automatic" but it's still worth documenting the automation.

Reply:
Done.